packages/govuk-frontend-review/src/common/nunjucks/index.mjs

```diff
@@ -20,6 +20,7 @@ export function renderer(app) {
   const env = nunjucksEnv(
     [join(paths.app, 'src/views')],
     {
+      dev: true, // log stack traces
```

Comment:
Ohhh, that's going to be handy for debugging! 🥳

Reply:
I discovered it recently and I'm getting bored of adding it 😆

You get to see Nunjucks stack traces right into global/filter code too